config/traits.yml

```diff
@@ -1450,8 +1450,10 @@ traits:
         decussate: When sequential pairs of opposite leaves attach at right angles to the pair above or below.
         distichous: When alternate or opposite leaves are arranged in two opposite rows.
         spiral: When alternate leaves are attached in a progressive spiral pattern along the stem.
-        fasciculate: A tight bundle or cluster of leaves attached to the stem in a single location, generally due to the shortening of internodes.
-        crowded: When internode length varies considerably along a stem, such that leaves appear crowded or tufted.
+        fasciculate: A tight bundle or cluster of leaves attached to the stem in a single location, generally due to the shortening of internodes. (Synonym, clustered)
+        crowded: When internode length varies considerably along a stem, such that leaves appear crowded or tufted. (Synonym, congested)
+        imbricate: XXX
```

Comment:
Just checking if imbricate's definition was meant to be filled in or not.

Reply:
they were all meant to be on a different branch - I hadn't caught that I'd mixed up branches